src/main/java/org/generationcp/middleware/service/impl/dataset/DatasetServiceImpl.java

```diff
@@ -1312,6 +1313,13 @@ public void setWorkbenchDataManager(final WorkbenchDataManager workbenchDataMana
 		return this.daoFactory.getExperimentDao().countObservationsPerInstance(datasetId);
 	}
 
+
+	@Override
+	public List<InstanceInfomationDTO> getInformationInstance(final Integer datasetId, final Integer studyId) {
```

Comment:
rename by getInstanceInformation or (getInstanceDetails or getInstanceMetadata if the class name is modified)